contracts/exchange/OrderHandler.sol

```diff
@@ -264,11 +263,30 @@ contract OrderHandler is IOrderHandler, BaseOrderHandler {
         }
     }
 
+    // @dev used by other handlers to avoid duplicating the same code on their side
+    // this method is similar to `executeOrder` but skips execution gas validation
+    function executeOrderForController(
```

Comment:
i think this should be named executeOrderFromController, to match e.g. DepositHandler.executeDepositFromController